modules/ibi-install-ibi-operator.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_content-type: PROCEDURE
+[id="ibi-install-ibi-operator_{context}"]
+= Installing the image-based installation Operator
+
+The Image Based Install Operator creates a configuration ISO using site-specific configuration resources that you define in the hub cluster. The IBI Operator boots the target host with the configuration ISO attached to begin the reconfiguration process. 
+
+The IBI Operator also validates the configuration resources that you define in the hub cluster, for example, the Operator checks YAML formatting and verifies networking configurations.
+
+[NOTE]
+====
+The IBI Operator is part of the multicluster engine for Kubernetes operator (MCE) from MCE version 2.10.
+====
+
+.Prerequisites
+
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster.
```

Comment:
Technically only MCE is required.

Reply:
You deployed a RHACM hub cluster or you deployed MCE.

Would this cover us @carbonin ?

Reply:
I think so.